DOCS/license-matching-guidelines-and-templates.md

```diff
@@ -103,18 +103,24 @@ Any hyphen, dash, en dash, em dash, or other variation should be considered equi
 
 Any variation of quotations (single, double, curly, etc.) should be considered equivalent.
 
-## B.7 Code Comment Indicators <a name="B.7"></a>
+## B.7 Code Comment Indicators or Separators <a name="B.7"></a>
 
 ### B.7.1  Purpose <a name="B.7.1"></a>
 
-To avoid the possibility of a non-match due to the existence or absence of code comment indicators placed within the license text, e.g. at the start of each line of text.
+To avoid the possibility of a non-match due to  the existence or absence of code comment indicators placed within the license text, e.g., at the start of each line of text, or repetitive characters to establish a separation of text, e.g., ---, ===, ___, or ***.
```

Comment:
Super-minor nit, looks like there's a stray extra space introduced by accident between "to" and "the"